### PR TITLE
feat!: Swapped default font family to Apercu

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### PR Checklist

- [x] Related to Abstract designs: https://www.figma.com/file/DIWmrLJ4iaesJH0mwvsiqV/Identity-Phase-1?node-id=1954%3A0
- [x] Related to JIRA ticket: GM-68
- [x] I have run this code to verify it works
- [x] This PR includes tests for the code change

### Description

Swaps the font family for our typography variables in SCSS and TypeScript to `Apercu`.

Keeps the old Regular Bold, Nunito Sans, and Ikona fonts around. I'll file a cleanup tasks for that.

### CODE REVIEW

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 1]
```diff
@@ -54,10 +51,10 @@ $line-height-headings: 1.1;
...
-$font-size-h3: 1.6rem;
-$font-size-h4: 1.4rem;
-$font-size-h5: 1.25rem;
-$font-size-h6: 1rem;
+$font-size-h1: px-rem(64);
+$font-size-h2: px-rem(44);
+$font-size-h3: px-rem(34);
+$font-size-h4: px-rem(26);
+$font-size-h5: px-rem(22);
+$font-size-h6: px-rem(20);
```
Comment (JoshuaKGoldberg) [inline packages/gamut-styles/utils/variables/_typography.scss:59]: @codecaaron do the font sizes in `typography.ts` also need to be updated?

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
 
-$font-family-apercu: "Apercu", -apple-system, BlinkMacSystemFont, "Segoe UI",
-  "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-  sans-serif;
-
-$font-family-suisse: "Suisse", "Apercu", -apple-system, BlinkMacSystemFont,
-  "Segoe UI", "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
-  "Helvetica Neue", sans-serif;
+// Todo: these should be deleted in a little bit, once everyone's done using them.
+// GM-89
```
Comment (JoshuaKGoldberg) [inline packages/gamut-styles/utils/variables/_typography.scss:18]: Todo: actually file a cleanup ticket

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.4-alpha.a01904.0<br/>@codecademy/gamut-styles@5.1.1-alpha.a01904.0<br/>@codecademy/gamut@14.0.4-alpha.a01904.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f43de6e7dc19c56e844409e--gamut-preview.netlify.app>https://5f43de6e7dc19c56e844409e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f43de6e7dc19c56e844409e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.194085.0<br/>@codecademy/gamut-styles@5.1.3-alpha.194085.0<br/>@codecademy/gamut@14.0.6-alpha.194085.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f45274ad26dbf8f502084bd--gamut-preview.netlify.app>https://5f45274ad26dbf8f502084bd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f45274ad26dbf8f502084bd>Deploy Logs</a></p>

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 3]
```diff
@@ -1,28 +1,25 @@
 // Font Families:
 
-$font-family-base: "Nunito Sans", -apple-system, BlinkMacSystemFont, "Segoe UI",
+$font-family-base: "Apercu", -apple-system, BlinkMacSystemFont, "Segoe UI",
   "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
   sans-serif;
 
-$font-base: $font-family-base;
```
Comment (kipprice) [inline packages/gamut-styles/utils/variables/_typography.scss:7]: is this var in use anywhere?

File: packages/gamut-styles/src/variables/typography.ts [hunk 4]
```diff
@@ -2,13 +2,13 @@ import { swatches as colorSwatch } from './colors';
...
 
-export const fontBase = `"Nunito Sans", -apple-system, BlinkMacSystemFont, "Segoe UI",
+export const fontBase = `"Apercu", -apple-system, BlinkMacSystemFont, "Segoe UI",
 "Roboto", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
 sans-serif`;
 
-export const fontHeadings = `"Regular Patch", "Regular Bold", "Nunito Sans",
--apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Ubuntu", "Cantarell",
-"Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif`;
+export const fontHeadings = `"Suisse", "Apercu", -apple-system, BlinkMacSystemFont,
```
Comment (kipprice) [inline packages/gamut-styles/src/variables/typography.ts:9]: I thought that we were going to use Aperçu for body + heading styles, and only use Suisse in specifically selected cases (at least to start). Has that changed?

File: packages/gamut-styles/utils/variables/_typography.scss [hunk 5]
```diff
@@ -54,10 +51,10 @@ $line-height-headings: 1.1;
...
-$font-size-h4: 1.4rem;
-$font-size-h5: 1.25rem;
-$font-size-h6: 1rem;
+$font-size-h1: px-rem(64);
+$font-size-h2: px-rem(44);
+$font-size-h3: px-rem(34);
+$font-size-h4: px-rem(26);
+$font-size-h5: px-rem(22);
+$font-size-h6: px-rem(20);
 $font-size-small: 85%;
```
Comment (kipprice) [inline packages/gamut-styles/utils/variables/_typography.scss:60]: where does this get used?

File: packages/gamut-styles/src/variables/typography.ts (hunk 4 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-styles/src/variables/typography.ts:9]: Aha, good catch, I didn't update this constant. That's correct.

File: packages/gamut-styles/utils/variables/_typography.scss (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-styles/utils/variables/_typography.scss:18]: https://codecademy.atlassian.net/browse/GM-102

File: packages/gamut-styles/utils/variables/_typography.scss (hunk 5 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-styles/utils/variables/_typography.scss:60]: `small`... and nowhere else. 😄

File: packages/gamut-styles/src/variables/typography.ts (hunk 4 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-styles/src/variables/typography.ts]: Huh @kipprice weird, I thought I pushed a change to this _before_ re-requesting review. It's pushed now. Sorry!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.9d1ce0.0<br/>@codecademy/gamut-styles@5.1.3-alpha.9d1ce0.0<br/>@codecademy/gamut@14.0.6-alpha.9d1ce0.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.3a727b.0<br/>@codecademy/gamut-styles@5.1.3-alpha.3a727b.0<br/>@codecademy/gamut@14.0.6-alpha.3a727b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f480dee3b7f79d954be43e3--gamut-preview.netlify.app>https://5f480dee3b7f79d954be43e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f480dee3b7f79d954be43e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f480e632b9aca8c0db5f35c--gamut-preview.netlify.app>https://5f480e632b9aca8c0db5f35c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f480e632b9aca8c0db5f35c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.f3f1bc.0<br/>@codecademy/gamut-styles@5.1.3-alpha.f3f1bc.0<br/>@codecademy/gamut@14.0.6-alpha.f3f1bc.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.41020a.0<br/>@codecademy/gamut-styles@5.1.3-alpha.41020a.0<br/>@codecademy/gamut@14.0.6-alpha.41020a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.fd4018.0<br/>@codecademy/gamut-styles@5.1.3-alpha.fd4018.0<br/>@codecademy/gamut@14.0.6-alpha.fd4018.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f4813b59405893b67ffd7de--gamut-preview.netlify.app>https://5f4813b59405893b67ffd7de--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f4813b59405893b67ffd7de>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.0.6-alpha.48e063.0<br/>@codecademy/gamut-styles@5.1.3-alpha.48e063.0<br/>@codecademy/gamut@14.1.0-alpha.48e063.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f48229a42500700d46a9ff2--gamut-preview.netlify.app>https://5f48229a42500700d46a9ff2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f48229a42500700d46a9ff2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@2.1.1-alpha.70e3c0.0<br/>@codecademy/gamut-styles@5.1.3-alpha.70e3c0.0<br/>@codecademy/gamut@15.1.0-alpha.70e3c0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f48249c334e3300bfb8a6b3--gamut-preview.netlify.app>https://5f48249c334e3300bfb8a6b3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f48249c334e3300bfb8a6b3>Deploy Logs</a></p>

**Pull request merged**